tests/tests_integration/test_api/test_diagrams.py

```diff
@@ -143,6 +143,7 @@ def test_run_diagram_detect_with_page_range(self, cognite_client):
         assert set(pages_with_annotations_per_subjob[1]) == {6, 11}
         assert set(pages_with_annotations_per_subjob[2]) == {50, 55}
 
+    @pytest.mark.skip(reason="Skip for now, raises 400: {'patternMode': ['Unknown field.']}")
```

Comment:
Nice

Reply:
This seem to have been resolved, so I'll remove the skip-mark